### PR TITLE
initial agent rebrand

### PR DESCRIPTION
**Pull request by bryanmacfarlane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Supported on Windows, OSX, Ubuntu and Red Hat.  Written for the .NET Core CLR as
 |![macOS](docs/res/apple_med.png) **macOS**|[![Build & Test][macOS-build-badge]][macOS-build]| 
 |![Linux](docs/res/linux_med.png) **Linux**|[![Build & Test][linux-build-badge]][linux-build]|
 
-[win-build-badge]: https://mseng.visualstudio.com/pipelinetools/_apis/build/status/VSTS.Agent.CI-Windows?branchName=master
+[win-build-badge]: https://mseng.visualstudio.com/pipelinetools/_apis/build/status/azure-pipelines-agent.ci-?branchName=master
```
Comment (azureDaveOps) [inline README.md]: I don't see that definition, should it be azure-pipleines-agent.ci-windows ?

Comment (azureDaveOps) [reply README.md]: https://dev.azure.com/mseng/PipelineTools/_apis/build/status/VSTS.Agent/azure-pipelines-agent.ci-windows?branchName=master
Now that branding is updated, we can also flip over to dev.azure.com/mseng

Comment (bryanmacfarlane) [reply README.md]: It's there:  https://dev.azure.com/mseng/PipelineTools/_build?definitionId=6916

Comment (bryanmacfarlane) [reply README.md]: I'll update the urls

Comment (bryanmacfarlane) [reply README.md]: fixed the missing -windows and tested all the urls

**Pull request merged**